playbooks/bios-upgrade-prod-dell.yaml

```diff
@@ -0,0 +1,74 @@
+- hosts: nerc_ocp_prod_workers
+  gather_facts: false
+  vars:
+    step_firmware: 2.70.70.70
+    target_firmware: 2.83.83.83
+    target_bios: 2.13.0
+    drm_server: 10.255.0.25
+    step_firmware_dir: fc430-step
+    target_firmware_dir: fc430
+    step_firmware_catalog: Catalog.xml
+    target_firmware_catalog: fc430_1.10_Catalog.xml
+
+  tasks:
+    - name: Get BMC IP
+      set_fact:
+        bmc_ip: "{{ lookup ('dig', '{{ bmc_addr }}') }}"
+
+    - name: Get firmware versions
+      delegate_to: localhost
+      dellemc.openmanage.idrac_firmware_info:
+        idrac_ip: "{{ bmc_ip }}"
```

Comment:
Out of curiosity, does this really require an ip address (that is, it really can't take a hostname)?

Reply:
It can work with the host name, but when you update to 2.83.83.83 you run into the host key checking issue and the iDRAC won't respond to it's host name. We'll eventually need the IP address to implement the host key checking workaround.

Reply:
Makes sense!